mlflow/server/js/src/utils/Utils.js

```diff
@@ -170,11 +171,15 @@ class Utils {
   }
 
   /**
-   * Renders the run name into a string.
+   * Renders the run name into a string, truncating it if necessary.
```

Comment:
Instead of truncating strings, it's better to use `text-overflow: ellipsis` on the components that hold this IMO (or some other overflow style that hides the stuff; even clipping if that's appropriate for tables). Right now, if you truncate the string, it won't be a fixed number of pixels because it depends on the size of characters in there, and moreover it might still be too big for some locations and too small for others. Let whoever includes the string deal with truncating it through CSS.

Reply:
Yeah avoiding one-implementation-fits-all logic makes sense, will update to handle truncation in the individual components.